### ISSUE TITLE
Message: default values for attributes showSummary and showDetail are not respected anymore

### ISSUE THREAD

**Issue by soma9000** [primefaces/primefaces#12623]
### Describe the bug

The attribute default values of `<p:message>` are `showSummary="false"` and `showDetail="true"` as of https://primefaces.github.io/primefaces/14_0_0/#/components/message This works fine in PF 13.0.x. In PF 14 the default values are not respected anymore. They seem to be the default values of `<p:messages>`.

![message-bug-pf-14 0 5](https://github.com/user-attachments/assets/eea71e5e-37ba-45e9-ba96-d87688c7fe77)

### Reproducer

[primefaces-test-master-message-bug.zip](https://github.com/user-attachments/files/17121832/primefaces-test-master-message-bug.zip)

1. unzip
2. mvn jetty:run
3. run page http://localhost:8080/
4. click button "create message"
5. expected behaviour and bugs are described

### Expected behavior

Behavior should be as documented and as in PF 13.

### PrimeFaces edition

Community

### PrimeFaces version

14.0.5

### JSF implementation

All

### JSF version

2.3, 4.0

### Java version

11

**Comment by melloware** [comment]
I will take a look and fix for 14.0.6 if that is the case

**Comment by melloware** [comment]
@soma9000 got to the bottom of it.  With this ticket:https://github.com/primefaces/primefaces/issues/10281 we refactored Messages and Growl to extend `javax.faces.component.UIMessages;` from Faces directly.

UIMessages in Faces has these defaults

```java
/**
     * <p>Return the flag indicating whether the <code>detail</code>
     * property of the associated message(s) should be displayed.
     * Defaults to false.</p>
     * 
     * @return <code>true</code> if detail is to be shown, <code>false</code> otherwise.
     */
    public boolean isShowDetail() {

        return (Boolean) getStateHelper().eval(PropertyKeys.showDetail, false);

    }

    /**
     * <p>Return the flag indicating whether the <code>summary</code>
     * property of the associated message(s) should be displayed.
     * Defaults to true.</p>
     * 
     * @return <code>true</code> if the summary is to be shown, <code>false</code> otherwise.
     */
    public boolean isShowSummary() {
        return (Boolean) getStateHelper().eval(PropertyKeys.showSummary, true);
    }
```

As you can see the Faces core defaults and this aligns PrimeFaces with the core Faces defaults.   So I think I need to update the documents and migration guide.

**Comment by soma9000** [comment]
Mmmmh, alignment to Faces sounds nice and changing the documentation would be a solution. But it has a big impact on existing code for the PrimeFaces users 'cause p:message should be used for nearly every input component... Maybe it's better to stay on the well established default values of p:message.

**Comment by melloware** [comment]
OK all 3 are inheriting UIMessages that is the issue.  Let me look at changing `Message`

**Comment by melloware** [comment]
@tandraschko is right its fixed for 14.0.6.  I was trying to be too clever with my original refactor.
